src/Tasks/ResolveSDKReference.cs

```diff
@@ -1056,11 +1056,11 @@ private void GetSDKManifestAttributes()
                 {
                     if (_sdkManifest.AppxLocations != null)
                     {
-                        foreach (string key in _sdkManifest.AppxLocations.Keys)
+                        foreach (KeyValuePair<string, string> kvp in _sdkManifest.AppxLocations)
                         {
-                            if (!AppxLocationsFromManifest.ContainsKey(key))
+                            if (!AppxLocationsFromManifest.ContainsKey(kvp.Key))
```

Comment:
What's the target dotnet framework? Dictionary.TryAdd could potentially be used)

Reply:
net472 and netstandard2.0 are among the target frameworks, so we can't, sadly.